src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString === '') {
+    return {};
+  }
+
+  const styleArray = sourceString.split(';');
```

Comment:
The variable name 'styleArray' could be more descriptive. While it does represent an array of styles, a name like 'styleDeclarations' might better reflect its contents.